stars/default/name.fab

```diff
@@ -256,7 +282,7 @@
  97938|ξ_Aql
  97675|ο_Aql
  97473|π_Aql
- 99742|ρ_Aql
+ 99742|ρ_Aql*
```

Comment:
Why asterisk added here?

Reply:
The punctuation "*" means this star originally in the neighbouring constellation, but it became part of the constellation it is now in with the official laying down of the constellation borders and proper motion.

Reply:
Please remove asterisk from designations, because this file contains modern designations.